frontend/app/components/work-packages/work-package-comment/work-package-comment.directive.ts

```diff
@@ -126,6 +128,13 @@ export class CommentFieldDirectiveController {
   public handleUserCancel() {
     this.editing = false;
     this.field.initializeFieldValue();
+    this.focusCommentField();
+  }
+
+  private focusCommentField() {
+    window.setTimeout(function(){
+      jQuery('.work-packages--activity--add-comment .inplace-editing--trigger-link')[0].focus();
```

Comment:
You don't need `element[0].focus()`, since jQuery exposes it. Still, please use the FocusHelper instead.